gitlab-ctl: add page (#4758)
tldr-pages/tldr: pages/common/gitlab-ctl.md

# gitlab-ctl

> > CLI tool for managing the GitLab omnibus.

Reviewer: ```suggestion
> A CLI tool for managing the GitLab omnibus.
```

> > A CLI tool for managing the GitLab omnibus.

Reviewer 2: Perhaps this will read better?
```suggestion
> CLI tool for managing the GitLab omnibus.
```

Author: @navarroaxel asked me to do the exact opposite in https://github.com/tldr-pages/tldr/pull/4758#discussion_r507657475 :stuck_out_tongue:

Reviewer: There are 13 pages in tldr using `A CLI...` :man_shrugging: Maybe we should fix them. @prudloff-insite please follow the suggestion from @sbrl.

> More information: <https://docs.gitlab.com/omnibus/maintenance/>.

> - Check the status of every service:

Reviewer: ```suggestion
- Display the status of every service:
```

`sudo gitlab-ctl status`

> - Check the status of a specific service:

Reviewer: ```suggestion
- Display the status of a specific service:
```

`sudo gitlab-ctl status {{nginx}}`

- Restart every service:

`sudo gitlab-ctl restart`

- Restart a specific service:

`sudo gitlab-ctl restart {{nginx}}`

> - Get the logs of every service:

Reviewer: The doc say this command is like `tail -f` so:
```suggestion
- Display the logs of every service and keep reading it until `Ctrl + C`:
```

> - Display the logs of every service and keep reading it until `Ctrl + C`:

Reviewer 2: Perhaps this will read better?
```suggestion
- Display the logs of every service and keep reading until `Ctrl + C` is pressed:
```

`sudo gitlab-ctl tail`

> - Get the logs of a specific service:

Reviewer: ```suggestion
- Display the logs of a specific service:
```
